test/Curve.js

```diff
@@ -147,26 +143,27 @@ contract("Curve", accounts => {
       [80000000000, 1500000000],
       [90000000000, 1500000000]
     ];
-    const gas = await Promise.all(
+    const gasChunks = await Promise.all(
       correct.map(async ([i, v]) => {
         const [neumarkUlps, gas] = await curveGas.curveGas.call(
           EUR_DECIMALS.mul(i)
         );
-        const neumarks = 0 | neumarkUlps.div(NMK_DECIMALS).floor().valueOf();
+        const neumarks = neumarkUlps.div(NMK_DECIMALS).floor().toNumber();
         assert.equal(neumarks, v, `Curve compute failed for value ${i}`);
-        return [i, 0 | gas.valueOf()];
+        return gas.toNumber();
       })
     );
-    const totalGas = gas.reduce((t, [_, gas]) => t + gas, 0);
-    console.log(`\t${correct.length} evaluations took ${gasCost(totalGas)}.`);
+    const totalGas = gasChunks.reduce((sum, gas) => sum + gas, 0);
+
+    expect(totalGas).to.be.eq(136282);
   });
 
   it("should issue Neumarks", async () => {
     assert.equal((await curve.totalEuroUlps.call()).valueOf(), 0);
     assert.equal((await neumark.totalSupply.call()).valueOf(), 0);
 
-    const r1 = await curve.issue(EUR_DECIMALS.mul(100), { from: accounts[1] }); // TODO check result
-    console.log(`\tIssue took ${gasCost(r1)}.`);
+    const r1 = await curve.issue(EUR_DECIMALS.mul(100), { from: accounts[1] });
+    expect(txGasCost(r1)).to.be.eq(191726);
```

Comment:
I like to have gas cost of tests as part of the test output, it is an import resource to be aware off. Compare it with the running time that mocha already prints.